src/lib/crypto/secp256k1.ts

```diff
@@ -291,6 +348,19 @@ const wrapSecp256k1Wasm = (
   const internalSigPtr = secp256k1Wasm.malloc(internalSigLength);
   const privateKeyPtr = secp256k1Wasm.malloc(privateKeyLength);
 
+  const internalRSigPtr = secp256k1Wasm.malloc(recoverableSigLength);
+  // tslint:disable-next-line:no-magic-numbers
+  const recoveryNumPtr = secp256k1Wasm.malloc(4);
+  // tslint:disable-next-line:no-bitwise no-magic-numbers
+  const recoveryNumPtrView32 = recoveryNumPtr >> 2;
+  
+  /*
+  const setRecoveryNumPtr = (value) => {
+    secp256k1Wasm.heapU32.set([value], recoveryNumPtrView32);
+  };
+  */
```

Comment:
Is this marking something needed for later? (Or can it be deleted?)

Reply:
I _thought_ I was going to use this later on, but the C recovery function takes in an actual int instead of a pointer, so there was never a need to set the value in the pointer, guess I forgot to delete it.